### DISCUSSION TITLE
How to prevent revalidating on focus change?

### DISCUSSION THREAD

**Discussion by OnkelTem** [nextauthjs/next-auth#6342] [Help]
Whenever I switch tabs in the browser or alt-tab to another application and back, it sends a new request to `/api/auth/session`.
I couldn't find anywhere in the docs how to prevent it.

Ideas?

**Comment by AK-101111** [comment]
I am having the exact same question.
@OnkelTem did you happen to find an answer to that?

**Comment by AK-101111** [comment]
I believe I found it. 
SessionProvider takes as a prop refreshOnWindowFocus which you can set it to false.
Mind you it will work on production, not dev.

`<SessionProvider  refetchOnWindowFocus={false}>`

there is also a refreshWhenOffline available.
documentation: https://next-auth.js.org/getting-started/client#options

**Reply by harshvats2000** [reply]
Works well in development.
